dts/bindings/pwm/nxp,imx-pwm.yaml

```diff
@@ -26,6 +26,22 @@ properties:
       description: |
         Enable for PWM to keep running in debug mode.
 
+    nxp,prescaler:
+      type: int
+      enum: [0, 1, 2, 3, 4, 5, 6, 7]
```

Comment:
Why not use a string enum here, and use `DT_ENUM_IDX` to get the integer value to use for the prescaler property? Something like this: https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/pinctrl/nxp%2Cmcux-rt-pinctrl.yaml#L79

Reply:
I didn't know that such macro. Fixed.